### PR TITLE
better timestamp for projects

### PR DESCRIPTION
**Pull request by laurelfan**
- don't show future timestamps
- update timespan as time passes (eg: a minute ago will change to 2 minutes ago in a minute)
- use jquery.timeago plugin (supports i18n as soon as we figure out the build pipeline)
- also changed 'Not saved' text to talk about clicking the run button

Unsaved:

Now click Run:

Wait 2 minutes:

### CODE REVIEW

**Comment by sfilman** [comment]
Looks good! @laurelfan can we have the string be "Click 'Run' to save" -- so capital 'C', and single quotes around the 'Run'

**Comment by laurelfan** [comment]
updated the string (see updated screenshot above)

**Comment by laurelfan** [comment]
updated the string, see updated screenshot above

File: shared/js/initApp.js [hunk 1]
```diff
@@ -163,22 +164,33 @@ function callbackSafe(callback, data) {
...
+        .append($('<span class="timestamp">').attr('title', dashboard.currentApp.updatedAt)).show();
+    $('.project_updated_at span.timestamp').timeago();
+  } else {
+    $('.project_updated_at').text("Click 'Run' to save"); // TODO i18n
+  } 
+}
+
 dashboard.saveProject = function(callback) {
-  $('.project_updated_at').text('Saving...'); // TODO (Josh) i18n
+  $('.project_updated_at').text('Saving...');  // TODO (Josh) i18n
```
Comment (joshlory) [inline shared/js/initApp.js]: Ah I totally forgot about this TODO :) I will fix.

File: shared/js/initApp.js [hunk 2]
```diff
@@ -163,22 +164,33 @@ function callbackSafe(callback, data) {
   }
 }
 
+dashboard.updateTimestamp = function() {
+  if (dashboard.currentApp.updatedAt) {
+    // TODO i18n
```
Comment (joshlory) [inline shared/js/initApp.js]: TODO

**Comment by joshlory** [comment]
LGTM!  If you want me to add the other i18n strings when I add 'Saving...' you can merge and I'll update them all at once.

**Comment by laurelfan** [comment]
cool, thanks for doing the i18n... hopefully the js build pipeline thing will solve that for the future...

**Pull request merged**